ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/workers-rhel8/sdn/provision/cucushift-installer-rehearse-aws-ipi-proxy-workers-rhel8-sdn-provision-chain.metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/proxy/workers-rhel8/sdn/provision/cucushift-installer-rehearse-aws-ipi-proxy-workers-rhel8-sdn-provision-chain.yaml",
```

Comment:
How about moving sdn to `aws/ipi/proxy/`, then we will have cucushift-installer-rehearse-aws-ipi-proxy-sdn-provision-chain.yaml, based that chain, we can add rhel-workers steps to have `cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-provision-chain.yaml`

Reply:
Similar suggestion to https://github.com/openshift/release/pull/31307/files#r954923552